tests/test_usage.py

```diff
@@ -77,13 +76,29 @@ def test_multicast_message_arg_main(self):
 			theResult = False
 		self.assertTrue(theResult, fail_fixture)
 
+	def test_multicast_hear_invalid_arg_main(self):
+		"""Tests the message argument for failure given invalid input"""
+		theResult = False
+		fail_fixture = str("""multicast.__main__.useTool(HEAR, junk) != 2""")
+		try:
+			with self.assertRaises(SystemExit):
+				self.assertIsNotNone(multicast.__main__.useTool("HEAR", ["--port", "test"]))
+				self.assertNotEqual(multicast.__main__.useTool("HEAR", ["--port", "test"]), 0)
+				self.assertNotEqual(multicast.__main__.useTool("HEAR", ["--port", "test"]), 1)
+			theResult = True
+		except Exception as err:
+			context.debugtestError(err)
+			self.fail(fail_fixture)
+			theResult = False
+		self.assertTrue(theResult, fail_fixture)
+
 	@unittest.expectedFailure
 	def test_multicast_hexdump_arg_main(self):
 		"""Tests the hexdump argument for failure given future tools"""
 		theResult = False
 		fail_fixture = str("""multicast.__main__.useTool(HEAR, hex) == error""")
 		try:
-			with self.assertRaises(NotImplementedError, fail_fixture):
+			with self.assertRaises(NotImplementedError):
```

Comment:
does this work
```suggestion
			with self.assertRaises(NotImplementedError, msg=fail_fixture):
```
so the message is kept?